README.md

```diff
@@ -11,3 +15,14 @@ for more information.
 
 Discuss features and ask questions on [LoopBack Forum](https://groups.google.com/forum/#!forum/loopbackjs).
 
+## Module Long Term Support Policy
+
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version | Status          | Published | EOL      |
+| ------- | --------------- | --------- | -------- |
```

Comment:
After a second thought, I wonder how that Apr / Dec 2019 come up? 
According to the node.js LTS schedule https://github.com/nodejs/Release, 
- when 1.x was out (jul 2014), the "current" Node.js version at that time should be 0.10. -> EOL already
- when 2.x was out (dec 2015), the "current" Node.js version at that time should be 4.x. -> EOL already
- when 3.x was out (feb 2017),  the "current" Node.js version at that time should be 6.x -> EOL April 2019. 

Are the LTS schedules proposed in this PR based on the fact that we should allow some time for users to update to a more current version so it is not exactly matching the Node.js LTS schedule?  If yes, I'm good with it.

Reply:
I was discussing EOL times with Chris and the rule for determining EOL is slightly different. We need to find all pairs of (Node version N.x, Module version M.x) that passed the following condition at some point in time: N.x was the latest LTS version of Node, M.x was the latest version of the module. For each of these pairs, M.x must be supported until N.x goes EOL.

So for example, because loopback-sdk-angular-cli 2.x was released in Dec 2015, after Node.js 4.x went LTS, we must assume that some people may have picked loopback-sdk-angular-cli 1.x to use with Node.js 4.x, an therefore loopback-sdk-angular-cli must be supported at least as long as Node.js 4.x is in LTS.

Here are the current LTS versions of Node.js:
 - 6.x in LTS from 2016-10-18
 - 8.x in LTS from 2017-10-31
 - 10.x in LTS from 2018-10-30

When Node.js 6.x entered LTS, the latest angular-cli version was 2.x, therefore we need to support 2.x until April 2019.

When Node.js 8.x entered LTS, the latest angular-cli version was 3.x, therefore we need to support 3.x until Dec 2019.

There is no need to support 1.x version any more.

Reply:
+1 - no need to support 1.x version